crashlog/startupreason.c

```diff
@@ -311,7 +311,8 @@ static void get_default_bootreason(char *bootreason) {
     unsigned int i;
     char bootreason_prop[PROPERTY_VALUE_MAX];
 
-    ret = get_cmdline_bootreason(bootreason_prop);
+    ret = property_get(PROP_BOOTREASON, bootreason_prop, "");
+    // ret = get_cmdline_bootreason(bootreason_prop);
```

Comment:
We can remove this line. Just add one comment before "property_get" like "Use property_get starting from xxx version".